src/scripts.js

```diff
@@ -11,22 +12,65 @@ let userDailyStepGoal = document.querySelector('.user-daily-step-goal-js');
 let userFriends = document.querySelector('.user-friends-js');
 let stepGoalComparison = document.querySelector('.step-goal-comparison-js');
 let waterConsumedByDay = document.querySelector('.water-consumed-by-day-js');
-let waterConsumedByWeek = document.querySelector('.water-consumed-by-week-js')
+let waterConsumedByWeek = document.querySelector('.water-consumed-by-week-js');
+let hoursSlept = document.querySelector('.hours-slept-js');
+let sleepQuality = document.querySelector('.sleep-quality-js');
+let lastWeekSleep = document.querySelector('.last-week-sleep-js');
+let recomendedSleep = document.querySelector('.sleep-percentage-js')
+let avgSleepQuality = document.querySelector('.avg-sleep-quality-js')
+let avgHoursSleptAllTime = document.querySelector('.all-time-avg-hours-slept-js')
+let parentDiv = document.querySelector('.parent')
+
+parent.addEventListener('click', makeItShake);
+
+function makeItShake(event) {
+  if (event.target.classList.contains('tile')) {
+    event.target.closest('.tile').classList.toggle('vibe')
+  }
+}
 
 displayUserInfo(user);
+displayWaterConsumtion();
```

Comment:
breaking down the massive function into smaller bites based on the different classes or data sets.  Nice